src/osp/OSPApplication.cpp

```diff
@@ -24,9 +24,11 @@
  */
 #include "OSPApplication.h"
 
+#include "osp/Resource/blueprints.h"
+
 using namespace osp;
 
-OSPApplication::OSPApplication()
+OSPApplication::OSPApplication() : m_glResources("gl", "gl-resources")
```

Comment:
I'd recommend doing this as an inline default.

E.g.

Package m_glResources{"gl", "gl-resources"};

where the variable is declared.